### PR TITLE
styling user avatar and fix some sass errors

### PR DESCRIPTION
**Pull request by goldipl**

### CODE REVIEW

File: resources/sass/core/_dark_header.scss [hunk 1]
```diff
@@ -52,6 +50,15 @@ body.theme-dark {
           }
 
           .media {
+            > a:first-of-type {
```
Comment (danon) [inline resources/sass/core/_dark_header.scss:53]: Sorry, but that is completely ambiguous to me. Looking at this style I have no idea what `a:first-of-type` and `> div` really are.

Could you add appropriate class names to these elements, please?

Comment (goldipl) [reply resources/sass/core/_dark_header.scss]: @danon Done

File: resources/sass/core/_dark_header.scss [hunk 2]
```diff
@@ -52,6 +50,15 @@ body.theme-dark {
           }
 
           .media {
+            .user-avatar-link {
```
[the remark this answers is not in the document]
Comment (goldipl) [reply resources/sass/core/_dark_header.scss]: @danon Done;)

**Comment by danon** [comment]
@goldipl Branch has unresolved conflicts. 

They happen to be in places where you removed commas.

**Pull request merged**